test/integration/scripts/execute.sh

```diff
@@ -26,7 +26,8 @@ go build cmd/edge_core.go
 #dynamically append testManager Module before starting integration test.
 sed -i 's/dbTest/dbTest, testManager/g' conf/modules.yaml
 #restart edge_core after appending testManager Module.
-pkill edge_core
+sudo pkill edge_core
+sleep 2s
```

Comment:
Please comment why we sleep 2s and is 2s enough?

Reply:
@m1093782566 , as suggested gave a comment on top.